src/cs/usfca/edu/histfavcheckout/externalapis/APIClient.java

```diff
@@ -52,7 +52,7 @@ public static SearchMoviesResponse getAllMovies(Set<Integer> movies) {
 		HttpURLConnection con = request.connect(url, "GET");
 		String response = request.getResponse(con);
 		if (response != null) {
-			//TODO: Log this: System.out.println("Received : " + response.toString());
+			System.out.println("Received : " + response.toString());
```

Comment:
As specified earlier. Let's only catch error workflows and not the success ones.

Reply:
So do I just comment this line out?

Reply:
You can actually remove it too.

Reply:
Okay got it